tests/pausing_test.go

```diff
@@ -53,136 +52,98 @@ import (
 )
 
 var _ = Describe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-compute]Pausing", decorators.SigCompute, func() {
+	const cirrosStartupTimeout = 90
 
-	var err error
 	var virtClient kubecli.KubevirtClient
-	const cirrosStartupTimeout = 90
 
 	BeforeEach(func() {
 		virtClient = kubevirt.Client()
 	})
 
 	Context("A valid VMI", func() {
-
 		var vmi *v1.VirtualMachineInstance
 
 		BeforeEach(func() {
 			vmi = libvmops.RunVMIAndExpectLaunch(libvmifact.NewCirros(), cirrosStartupTimeout)
 		})
 
-		When("paused via API", func() {
-			It("[test_id:4597]should signal paused state with condition", func() {
+		It("[test_id:4597]should signal paused state with condition", func() {
+			By("Pausing VMI")
+			err := virtClient.VirtualMachineInstance(vmi.Namespace).Pause(context.Background(), vmi.Name, &v1.PauseOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Eventually(matcher.ThisVMI(vmi), 30*time.Second, time.Second).Should(matcher.HaveConditionTrue(v1.VirtualMachineInstancePaused))
 
-				err = virtClient.VirtualMachineInstance(vmi.Namespace).Pause(context.Background(), vmi.Name, &v1.PauseOptions{})
+			By("Unpausing VMI")
+			err = virtClient.VirtualMachineInstance(vmi.Namespace).Unpause(context.Background(), vmi.Name, &v1.UnpauseOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Eventually(matcher.ThisVMI(vmi), 30*time.Second, time.Second).Should(matcher.HaveConditionMissingOrFalse(v1.VirtualMachineInstancePaused))
+		})
+
+		It("[test_id:3225]should signal paused state with condition when paused multiple times", func() {
```

Comment:
Do you know why we need to start/stop multiple times?

What is the value of `test_id:4597` if we repeat it three times in `test_id:3225`?

Reply:
> Probably to rule out any flakes.

I dont' understand you: repeating a raceful test three times would only increase the probability of the race occurring.

> I was thinking about dropping it. Would you agree?

I don't know. Luckily we have @stu-gott around, maybe he remembers. In any case, I'd do it only in a separate commit, with a very clear commit message.

Reply:
> I dont' understand you: repeating a raceful test three times would only increase the probability of the race occurring.

Exactly, the test is run three times in succession to ensure it passes every time. That is what the `check-tests-for-flakes` lane does too. All modified tests are run five times to ensure they pass every time. Please correct me if I'm wrong.

Reply:
I don't get why pause/unpause is different than any other test that it needs this repetition.